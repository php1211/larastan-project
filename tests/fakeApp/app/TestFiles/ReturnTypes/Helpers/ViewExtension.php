<?php

declare(strict_types=1);

namespace App\TestFiles\ReturnTypes\Helpers;

use Illuminate\View\View;
use Illuminate\Http\JsonResponse;
use Illuminate\Contracts\View\Factory;

class ViewExtension
{
    public function testSimpleView(): View
    {
        return view('test');
    }

    public function testFactory(): Factory
    {
        return view();
    }

    public function testFail(): JsonResponse
    {
        return view('test');
    }
}
