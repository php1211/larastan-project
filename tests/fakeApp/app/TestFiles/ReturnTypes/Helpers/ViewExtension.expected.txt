23:Method App\TestFiles\ReturnTypes\Helpers\ViewExtension::testFail() should return Illuminate\Http\JsonResponse but returns Illuminate\View\View.
